@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getInput.sqf

```diff
@@ -41,6 +41,14 @@ private _utilLevel_pct     = _heli getVariable "fza_systems_utilLevel_pct";
 private _accOn             = _heli getVariable "fza_systems_accOn";
 private _apuOn             = _heli getVariable "fza_systems_apuOn";
 
+if (gunner _heli isEqualTo player) exitWith { 
```

Comment:
That doesn't mean all outputs are nulled no? Surely we want to make sure no addForces are performed?

Also does this not allow passing back and forth?

Does this constantly systemchat into the gunner's seat?

Reply:
I've got this configured with the simplest possible case that of the the gunner gets ZERO control, and it system chats out so we see that it's working. All of that will be removed once it's validated, but I need visibility in both seats to see that it's working. It'll be a quick fix to remove the system chats and then add additional functionality. For now, I just want to verify that this 100% locks the front seat out entirely from any kind of input being put into the system. You can't isolate the torque from the input because of how the system works, so you just have to null out the gunners flight control inputs entirely.